worker-markup-core/src/main/java/com/github/cafdataprocessing/worker/markup/core/EmailSplitter.java

```diff
@@ -132,8 +129,17 @@ public void generateEmailTags(Document doc) throws JDOMException, ExecutionExcep
 
     private List<String> getSeparatedEmails(String emailContent) throws JDOMException, ExecutionException, InterruptedException
     {
-        List<Integer> indexes = jepExec.getMessageIndexes(emailContent);
-        return separatedIndividualMessages(emailContent, indexes);
+        List<Integer> emailStartLineNumbers = new ArrayList<>();
+        final String lineMarkers = TalonEmailSplitter.splitEmails(emailContent);
+        char[] markers = ((String) lineMarkers).toCharArray();
+        int size = markers.length;
+        for (int i = 0; i < size; i++) {
+            if (markers[i] == 's') {
+                emailStartLineNumbers.add(i);
+            }
+        }
```

Comment:
Lines 132-138 have been copied from:

https://github.com/CAFDataProcessing/util-email-content-segregation/blob/develop/src/main/java/com/hpe/caf/worker/emailsegregation/ContentSegregation.java#L95

If these changes all work, I think we should be able to archive the following repos:

- https://github.com/CAFDataProcessing/util-email-content-segregation
- https://github.com/CAFDataProcessing/util-email-content-segregation-script
- https://github.com/CAFapi/opensuse-jeptalon-image